javascript/trees/breadthFirst/README.md

```diff
@@ -0,0 +1,12 @@
+# Challenge Summary
+Write a breadth-first array traversal function and output the value of the source tree's nodes as an array.
+
+## Whiteboard Process
+![Whiteboard_1stDraft](/javascript/trees/breadthFirst/cChall17_Whiteboard_1stDraft.jpg)
+This initial board is incorrect. I realized as I was finishing the visualization that while the output will represent the nodes in a top-down, left-to-right order, the way to get there actually requires the node values to be pushed into the queue from the bottom up. I'm out of time to fix it tonight but will rework it tomorrow and resubmit with more progress.
```

Comment:
That alright. Part of the visualization is to help you understand what is being asked.